base-utils/bzip2/autobuild/prepare

```diff
@@ -1,4 +1,24 @@
-sed -e 's/^CFLAGS=\(.*\)$/CFLAGS=\1 \$(BIGFILES)/' -i ./Makefile-libbz2_so
+abinfo 'Patching the CFLAGS in Makefile of libbz2.so...'
+sed -e 's/^CFLAGS=\(.*\)$/CFLAGS=\1 \$(BIGFILES)/g' \
+    -i "$SRCDIR"/Makefile-libbz2_so
 
-sed -i "s|-O2|${CFLAGS}|g" Makefile
-sed -i "s|-O2|${CFLAGS}|g" Makefile-libbz2_so
+abinfo 'Replacing CFLAGS in Makefiles with what Autobuild3 defined...'
+sed -e "s|-O2|${CFLAGS}|g" -i "$SRCDIR"/Makefile
+sed -e "s|-O2|${CFLAGS}|g" -i "$SRCDIR"/Makefile-libbz2_so
+
+abinfo 'Generating bzip2.pc with HERE documents (contents refer from Arch Linux)...'
+cat << EOF > "$SRCDIR"/bzip2.pc
+prefix=/usr
+exec_prefix=/usr
+bindir=\${exec_prefix}/bin
+libdir=\${exec_prefix}/lib
+includedir=\${prefix}/include
+
+Name: bzip2
+Description: A file compression library
+Version: $PKGVER
+Libs: -L\${libdir} -lbz2
+Cflags: -I\${includedir}
+EOF
+
+cat "$SRCDIR"/bzip2.pc
```

Comment:
You probably want to clean this up, since usually, you don't need to inspect the contents of the generated file.

```suggestion

```